snap/snapcraft.yaml

```diff
@@ -0,0 +1,35 @@
+name: beets
+base: core20
+summary: Beets is the media library management system for obsessive music geeks.
+description: |
+  The purpose of beets is to get your music collection right once and for all.
+  It catalogs your collection, automatically improving its metadata as it goes.
+  It then provides a bouquet of tools for manipulating and accessing your music.
+confinement: strict
+adopt-info: beets
+
+apps:
+  beet:
+    command: bin/beet
+    plugs:
+      - home
+      - removable-media
+      - network
+      - network-bind
+
+parts:
+  beets:
+    source: .
+    override-pull: |
+      snapcraftctl pull
+      version="$(git describe --always --tags| sed -e 's/^v//;s/-/+git/;y/-/./')"
+      [ -n "$(echo $version | grep "+git")" ] && grade=devel || grade=stable
+      snapcraftctl set-version "$version"
+      snapcraftctl set-grade "$grade"
+    plugin: python
+    python-packages:
```

Comment:
are these supposed to be the dependencies of the project? This list seems different than what's in `setup.py`

Reply:
These are extras_require in setup.py, for which there is no declarative way in Snapcraft to specify yet, but I can migrate to that once it is supported. I only considered some plugins and not all, perhaps I should just consider supporting all plugins.

Reply:
Interesting! It's hard to say what to do if there's no room for optional dependencies… another option would be to include _no_ plugin dependencies and just leave it at that for now?